yellowbrick/classifier/classification_report.py

```diff
@@ -215,7 +232,8 @@ def classification_report(model, X, y=None, ax=None, classes=None, **kwargs):
     visualizer = ClassificationReport(model, ax, classes, **kwargs)
 
     # Create the train and test splits
-    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2,
+                                                        random_state=42)
```

Comment:
Ah, I see what happened - I meant add random state as an argument to this function (Default=None), then pass it to the `train_test_split`function, that way the user can supply the random state if required (e.g. for testing).